addons/diagnostic/xeh/config.cpp

```diff
@@ -2,6 +2,9 @@
 #ifndef EH_DEBUG_ENABLED
     class CfgPatches {
         class cba_diagnostics_xeh_disabled {
+            author = "$STR_CBA_Author";
+            name = "$STR_CBA_Optional_Component";

```

Comment:
I added it everywhere I can, but there are some PBOs that don't or can't include the script_mod